src/client/datascience/devTools/jupyterOutputChannel.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { window } from 'vscode';
+import { IOutputChannel } from '../../common/types';
+
+/**
+ * Returns the output panel for output related to Jupyter Server.
+ */
+export function getJupyterOutputChannel(isDevMode: boolean, defaultOutputChannel: IOutputChannel): IOutputChannel {
+    if (!isDevMode) {
+        return defaultOutputChannel;
+    }
+    // This isn't added to list of disposables, that should be fine (only used in dev mode).
+    const jupyterServerOutputChannel = window.createOutputChannel('Dev: Jupyter Server');
```

Comment:
No, not displayed to end users at all.
Only used for development purposes.

Reply:
I don't want users to have two output panels, else getting information from users for github issues introduces a lot more steps for users... they'd provide output for one & not the other, etc.

Reply:
This output panel is a dedicated output panel to display output from Jupyter Server, for debugging purposes.
Else its very time consuming to go through the existing output panel, with all of the logging, difficult to tell what came from Jupyter server vs what we logged, etc & have to menatally join strings to see what came through..

Reply:
Happy to remove this, 
This is dev output panel i brought up in a standup/or triage meeting.

Reply:
> No it's a good idea. Not sure why we wouldn't want to show it to users too. When we have a jupyter error, we point them at the log. We could point them at this instead/in addition to the log.

Will create a separate issue for that.
I'm concerned this could complicate (confuse users) our process of getting information from users.